blockreassurance.php

```diff
@@ -165,8 +201,13 @@ public function install()
      */
     public function uninstall()
     {
-        include_once(dirname(__FILE__) . '/sql/uninstall.php');
+        // SQL
+        $sql = 'DROP TABLE IF EXISTS `' . _DB_PREFIX_ . 'psreassurance`, `' . _DB_PREFIX_ . 'psreassurance_lang`';
```

Comment:
What if we already delete the table by mistake?
This one could be optional :thinking:

Reply:
@PierreRambaud The "IF EXISTS" permits to check if the table exists always.

Reply:
Oh, didn't see that :D